exist-core/src/test/xquery/dates/convert-dates.xqm

```diff
@@ -25,68 +25,68 @@ module namespace fd="http://exist-db.org/xquery/test/convert-dates";
 
 declare namespace test="http://exist-db.org/xquery/xqsuite";
 
-(: verify that we can convert xs:string to :xs:dateTimeStamp  :)
+(: verify that we can convert xs:string to :xs:dateTime :)
 declare
     %test:args("2012-06-26T23:14:22.566+02:00")
     %test:assertEquals("11:14 pm on Tuesday, June 26th, 2012")
-function fd:format-dateTimeStamp($date as xs:dateTimeStamp) {
+function fd:format-dateTime($date as xs:dateTime) {
```

Comment:
This test was valid in its original form - testing `xs:dateTimeStamp`.

Reply:
This has also now been undone.